models/todo.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const today = new Date()
+let yesterday = new Date(today)
+yesterday.setDate(yesterday.getDate() - 1)
+yesterday = yesterday.toString()
+
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Todo extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  };
+  Todo.init({
+    title: DataTypes.STRING,
+    desc: DataTypes.STRING,
+    status: DataTypes.STRING,
+    due_date: {
+      type: DataTypes.DATE,
+      validate: {isAfter: yesterday}
+    }
```

Comment:
kasih validasi in case user masukin data kosong atau string kosong

Reply:
done